airbyte-integrations/connectors/source-gnews/source_gnews/gnews.yaml

```diff
@@ -1,5 +1,107 @@
 version: "0.1.0"
 
+schemas:
```

Comment:
nit, but can we move the schemas to the bottom of the file? i think the definitions and streams are likely to be the parts of the manifest that change most, so it's more intuitive to have them at the top.

Reply:
@brianjlai I'm not super familiar with yaml but am not seeing a way to do forward referencing. From what I've found the anchor needs to come before the reference. Or are you aware of a syntax that I can use to do this?

Reply:
ah sorry you are correct and my mistake. I thought we did some preparsing to allow for forward references even if defined after, but doesn't seem like we do, so it's fine to have this at the top.

Reply:
No worries (sorry, deleted my comment because I thought maybe I'd spoken prematurely and was looking more closely at the issue). Going to write up an issue for updating the parser to handle forward references, since I do think that will make the yaml more readable.